tests/integration/test_vulnerability_detector/test_feeds/alas/test_extra_tags_alas_feed.py

```diff
@@ -99,9 +195,66 @@ def test_no_feed_changes(copy_files, clean_vuln_tables, after_execution_clean_vu
 
 
 @pytest.mark.parametrize('test_values', vd.EXTRA_TEST_VALUES, ids=vd.EXTRA_TEST_IDS)
-def test_extra_tags_alas_linux_feed(copy_files, modify_feed, daemons_handler, test_values, set_system, get_configuration, configure_environment,
-                                    restart_modulesd_function, file_monitoring):
-    """Check if Vulnerability Detector behaves as expected while importing Amazon Linux JSON feed with extra tags."""
+def test_extra_tags_alas_linux_feed(copy_files, modify_feed, daemons_handler, test_values, set_system,
+                                    get_configuration, configure_environment, restart_modulesd_function,
+                                    file_monitoring):
+    '''
+    description: Check if Vulnerability Detector behaves as expected while importing Amazon Linux JSON feed with extra
+                 tags. To do this, a vulnerability event is launched and check if the number of vulnerabilities inserted
+                 in the VULNERABILITIES table of CVE DB is the expected and finally check if the process is still
+                 running.
+
+    wazuh_min_version: 4.2.0
```

Comment:
```suggestion
    wazuh_min_version: 4.3.0
```

Reply:
Done in this [link](https://github.com/wazuh/wazuh-qa/commit/c02bcd31ed6d3d0f57d519bdc2051b8eafeede60#:~:text=c02bcd31ed6d3d0f57d519bdc2051b8eafeede60)